src/dbnode/storage/database_test.go

```diff
@@ -704,7 +704,6 @@ func TestDatabaseUpdateNamespaceSchemaNotSet(t *testing.T) {
 	mapCh <- nsMap
 
 	// wait till the update has propagated
-	<-updateCh
 	time.Sleep(10 * time.Millisecond)
```

Comment:
does this sleep do anything anymore?

Reply:
<-updateCh is flaky, sometimes it block forever
I found that wait for 10ms is good enough for etcd change to show up.
If we fix the flaky <-updateCh, then we can remove the sleep.
 
Either block wait, or time wait, we don't need both.